framework/source/class/qx/util/ResourceManager.js

```diff
@@ -291,8 +298,16 @@ qx.Class.define("qx.util.ResourceManager",
           }
           // If the resourceUri begins with a single slash, include the current
           // hostname
-          else if (resourceUri.match(/^\//) != null) {
-            statics.__urlPrefix[lib] = window.location.protocol + "//" + window.location.host;
+          else if (resourceUri.match(/^\//) != null)
```

Comment:
I have no further comments on formatting as long as you don't try to "fix" it (unless, of course, you fix it to be the way *I* like it.) :-)

As previously stated, I don't really understand what this PR is doing with 'base', so I'm not voting on the technical merits of this one.